go/type.go

```diff
@@ -214,15 +218,18 @@ func (self *StringReader) __init__(lines []string) {
 					self.pos = append(self.pos, pos{lnum: lnum + 2, col: col + 1})
 				}
 				col += len(c)
+				offset += len(c)
 			}
 			lnum += 1
+			offset += 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should replace offset += 1 with offset++